src/pages/task/actionButtons/previousCommits/index.tsx

```diff
@@ -51,204 +36,138 @@ export const PreviousCommits: React.FC<PreviousCommitsProps> = ({ taskId }) => {
+    <Tooltip
+      justify="middle"
+      trigger={
+        <Button size={Size.Small} disabled>
+          Previous commits
+        </Button>
       }
-    }
-  }, [versionMetadata]); // eslint-disable-line react-hooks/exhaustive-deps
-
-  // Hook that triggers fetching the last passing task if it needs to be fetched.
-  useEffect(() => {
-    if (!hasFetchedLastPassing && shouldFetchLastPassing) {
-      fetchLastPassing({
-        variables: {
-          projectIdentifier,
-          skipOrderNumber,
-          buildVariantOptions: {
-            ...bvOptionsBase,
-            statuses: [TaskStatus.Succeeded],
-          },
-        },
-      });
-    }
-  }, [shouldFetchLastPassing]); // eslint-disable-line react-hooks/exhaustive-deps
-
-  // Hook that triggers fetching the last executed task if it needs to be fetched.
-  useEffect(() => {
-    if (!hasFetchedLastExecuted && shouldFetchLastExecuted) {
-      fetchLastExecuted({
-        variables: {
-          projectIdentifier,
-          skipOrderNumber,
-          buildVariantOptions: {
-            ...bvOptionsBase,
-            statuses: finishedTaskStatuses,
-          },
-        },
-      });
-    }
-  }, [shouldFetchLastExecuted]); // eslint-disable-line react-hooks/exhaustive-deps
-
-  const Link = useLGButtonRouterLink(link);
-
-  return (
-    <PreviousCommitsWrapper>
-      <StyledSelect
-        size="small"
-        data-cy="previous-commits-select"
-        label="Previous commits for this task"
-        allowDeselect={false}
-        onChange={(v: CommitType) =>
-          dispatch({ type: "setSelectState", selectState: v })
+    >
+      No previous versions available.
+    </Tooltip>
+  ) : (
+    <Menu trigger={<Button size={Size.Small}>Previous commits</Button>}>
+      <MenuItem
+        as={Link}
+        disabled={parentLoading}
+        onClick={() =>
+          sendEvent({
+            name: "Submit Previous Commit Selector",
+            type: CommitType.Base,
+          })
         }
-        value={selectState}
-        disabled={!versionMetadata?.baseVersion}
+        to={linkObject[CommitType.Base]}
       >
-        <Option value={CommitType.Base}>
-          Go to {versionMetadata?.isPatch ? "base" : "previous"} commit
-        </Option>
-        <Option value={CommitType.LastPassing}>
-          Go to last passing version
-        </Option>
-        <Option value={CommitType.LastExecuted}>
-          Go to last executed version
-        </Option>
-      </StyledSelect>
-
-      <ConditionalWrapper
-        condition={disableButton}
-        wrapper={(children) => (
-          <Tooltip
-            align="top"
-            justify="middle"
-            triggerEvent="hover"
-            trigger={children}
-          >
-            {loading
-              ? `Fetching...`
-              : `There is no version that satisfies this criteria.`}
-          </Tooltip>
-        )}
+        {versionMetadata?.isPatch ? "Base" : "Previous"} commit
```

Comment:
I think that it's not obvious that these options send you to a different page, which I think could be fixed by just tweaking the menu options to start with "Go to..." or "View..."

<img width="245" alt="Screenshot 2023-11-01 at 1 04 00 PM" src="https://github.com/evergreen-ci/spruce/assets/47064971/23493746-a047-4ae5-95c9-23b46d6d499a">